straxen/plugins/bayes_peaklet_classification.py

```diff
@@ -0,0 +1,141 @@
+import numpy as np
+import numba
+from scipy.special import logsumexp
+import strax
+import straxen
+
+
+class BayesPeakletClassification(strax.Plugin):
+    """
+    Bayes Peaklet classification
+    Returns the ln probability of a each event belonging to the S1 and S2 class.
+    Uses conditional probabilities and data parameterization learned from wfsim data.
+
+    :param peaklets: peaklets
+    :param waveforms: peaklets waveforms in PE/ns
+    :param quantiles: quantiles in ns
+    :returns: the ln probability of a each peaklet belonging to S1 and S2 class
+    """
+    provides = 'peaklet_classification_bayes'
+    depends_on = ('peaklets',)
+    __version__ = '0.0.1'
+    dtype = (strax.peak_interval_dtype
+             + [('s1_ln_prob', np.float32, 'S1 ln probability')]
+             + [('s2_ln_prob', np.float32, 'S2 ln probability')]
+            )
+
+    # Descriptor configs
+    bayes_config_file = straxen.URLConfig(
+        default='resource://'
+        'conditional_probabilities_and_bins_v1_w_global_v6.npz?fmt=npy',
+        help='Bayes configuration file, conditional probabilities tables and Bayes discrete bins'
+    )
+    num_nodes = straxen.URLConfig(
+        default=50,
+        help='Number of attributes(features) per waveform and quantile'
+    )
+    classes = straxen.URLConfig(
+        default=2,
+        help='Number of label classes S1(1)/S2(2)'
+    )
```

Comment:
Maybe keep in mind that these config names like `'classes'` are visible to all other plugins, as such, I would advice making it a bit more explicit: `classes->n_bayes_classes / bayes_peak_types`, `bayes_nodes` or something like it?

Reply:
good point, fixed